restore.go

```diff
@@ -171,7 +172,23 @@ func createContainer(backup Backup) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	// io.Copy(os.Stdout, reader)
+
+	_, err = cli.ContainerInspect(ctx, name)
+	if err == nil {
```

Comment:
See my comment on #23: try to keep the happy path on the left edge. Check for an error and bail out immediately, then keep the rest of the code on the left edge.

Related: what happens in case of an error here? Shouldn't we return it?